packages/hyperion-util/src/PersistentData.ts

```diff
@@ -15,50 +17,73 @@ function getStorage(storageName: 'sessionStorage' | 'localStorage'): Storage | n
     storage.removeItem(x);
     return storage;
   } catch (e) {
-    return null;
+    return {
```

Comment:
What's the motivation for this change?  It's weirdly expensive no?

Reply:
Actually this to remove an unlikely null check in the expensive part. See the `save()` method bellow. 
The `gteStorage` almost always succeeds, but because of the signature, we might return null, which means later on we have to always say `this.storage?.setItem`. By not returning null, the extra null check can go away.

Reply:
Would just returning a frozen object be overall faster? Or at least a value prop rather than a getter setter pair.

Reply:
what do you mean by getter/setter? This is the same interface that is used in the code bellow. I could turn it into a singleton, but as of now, it is called may be a few times total. so, didn't want to add the complexity for singleton.